### DISCUSSION TITLE
Data Model: Multiple half-width fields next to one other half-width field

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#14089] [Feature Requests]
I hope its not a duplicate because I found it hard to search for it.

What I would like is the ability to place multiple half-width fields next to one bigger half-width field.

For example I have a huge text field on the left side and on the right side I have an image field and some config fields.

<table>
<tr>
<th>How it is</th>
<th>What I want</th>
</tr>

<tr>
<td>

![image](https://user-images.githubusercontent.com/10547444/175550925-3160c486-daa8-4a6b-a0af-6b4a49e05e42.png)

</td>

<td>

![image](https://user-images.githubusercontent.com/10547444/175551270-7df6754d-d7ce-46ea-a18e-b817c096e16b.png)

</td>
</tr>
</table>

## Possible idea how to implement:
My idea would be to allow the `raw group` field to be used as half with.
Then one could place the raw group field next to a text field and add the image and input field inside.

**Comment by rijkvanzanten** [comment]
The app layout engine already supports this! We've historically disabled the width selection in data model to prevent confusion, but if you hack around that by setting width=half in the database directly, this works as expected:

<img width="1043" alt="CleanShot 2022-06-24 at 10 07 23@2x" src="https://user-images.githubusercontent.com/9141017/175552743-f667d660-dfae-42f0-9af4-8951124a667b.png">

<img width="1062" alt="CleanShot 2022-06-24 at 10 07 31@2x" src="https://user-images.githubusercontent.com/9141017/175552772-624ba1f7-2217-4770-9eae-f7f612625365.png">

Should we just enable those width options for groups in data model @benhaynes?

![CleanShot 2022-06-24 at 10 07 57@2x](https://user-images.githubusercontent.com/9141017/175552909-dbcc2904-e8f4-443f-abed-33bb1856bac7.png)

**Reply by benhaynes** [reply]
I'd rather consider all this when we have time to do a full sprint on the Form Layout option upgrades. If y'all want to override me here (I know Rijk has wanted this for a while), that's fine... but I think it can (and will) lead to some messy/bad form designs. I'd like to keep the "training wheels" on a bit until we have thought through ways to make this work with things like vertical height (set/flex) built into the form designer.

**Reply by sxsws** [reply]
This is great, and I remember starting a discussion about allowing groups to be half width a while back! https://github.com/directus/directus/discussions/13378

@benhaynes You're right in that there needs to be some sort of 'container' to place fields inside of to allow neater groupings

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
